msal-dotnet-articles/acquiring-tokens/desktop-mobile/wam.md

```diff
@@ -1,63 +1,85 @@
+> With the introduction of the updated broker, we have also updated MSAL.NET to expose its capabilities under a simplified API. Starting with version 4.52.0, the new broker will be the default in MSAL.NET.
 
-The improved WAM broker fixes a number of issues with the legacy WAM implementation and provides other benefits:
+The new WAM fixes a number of issues with the legacy implementation and provides other benefits, including:
 
 - New implementation is more stable, easier to add new features, and has less chance of regressions.
-- Works in apps that are run-as-admin.
-- Adds support for Proof-of-Possession tokens.
-- Fixes assembly size issues.
+- Works in apps that are ran under the Administrator user context.
+- Adds support for [Proof-of-Possession tokens](../../advanced/proof-of-possession-tokens.md).
+- Decreases assembly size.
 
 ## Enabling WAM
 
-Due to platform-specific and backwards compatibility requirements, WAM implementation resides in three packages:
+Due to platform-specific and backwards compatibility requirements, WAM support is split across three packages:
 
-- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/)(i.e. MSAL)
-- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) - WAM support
-- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) - WAM and WebView2 support
+- [Microsoft.Identity.Client](https://www.nuget.org/packages/Microsoft.Identity.Client/) (i.e., MSAL) - core library for token acquisition.
+- [Microsoft.Identity.Client.Broker](https://www.nuget.org/packages/Microsoft.Identity.Client.Broker/) - adds support for authentication with the broker.
+- [Microsoft.Identity.Client.Desktop](https://www.nuget.org/packages/Microsoft.Identity.Client.Desktop/) - adds support for authentication with the broker on legacy platforms, like .NET Framework, .NET Core 3.1, and UWP.
 
-After referencing the correct packages, call `WithBroker(BrokerOptions)` with options and pass a window handle.
+After referencing the relevant packages, call [`WithBroker(BrokerOptions)`](xref:Microsoft.Identity.Client.Desktop.WamExtension.WithBroker*) with broker configuration options and [a window handle](#parent-window-handles) that the broker will be bound to.
 
 ```csharp
-var pca = PublicClientApplicationBuilder.Create(CLIENT_ID)
-            .WithAuthority("https://login.microsoftonline.com/common")
-            .WithBroker(new BrokerOptions(BrokerOptions.OperatingSystems.Windows))
-            .WithParentActivityOrWindow(() => WINDOW_HANDLE)
-            .Build();
+BrokerOptions options = new BrokerOptions(BrokerOptions.OperatingSystems.Windows);
+
+options.Title = "My Awesome Application";
+options.ListOperatingSystemAccounts = true;
+
+IPublicClientApplication app =
```

Comment:
adding authority and adding a link guidance [here](https://learn.microsoft.com/en-us/azure/active-directory/develop/msal-client-application-configuration), will help with teams adopting to use WAM